workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/EntryPoint.cs

```diff
@@ -60,12 +61,17 @@ public int Run()
             var bundlePath = GenerateBundle();
             var schemaBundle = SchemaBundle.LoadBundle(File.ReadAllText(bundlePath));
             var store = new DetailsStore(schemaBundle, options.SerializationOverrides);
-            var workerGenerationJob = new WorkerGenerationJob(options.NativeOutputDirectory, options, fileSystem);
-            var singleJob = new SingleGenerationJob(options.NativeOutputDirectory, store, fileSystem);
 
-            var runner = new JobRunner(fileSystem);
+            var jobs = AppDomain.CurrentDomain
```

Comment:
This should probably guard against the `ReflectionTypeLoadException` and others.
Since the assembly might have loaded third-party dll's that mess with this.